src/billforward.js

```diff
@@ -1207,19 +1207,26 @@
             if (!payload) {
                 var $form = $(self.transaction.formElementCandidate);
 
-                payload = {
-                    accountID: self.transaction.accountID,
-                    routingNumber: bfjs.core.getFormValue("routingNumber", $form),
-                    accountNumber: bfjs.core.getFormValue("accountNumber", $form),
-                    stripeCustomerID: bfjs.core.getFormValue("stripeCustomerID", $form),
-                    holderName: bfjs.core.getFormValue("holderName", $form),
-                    bankAccountName: bfjs.core.getFormValue("bankAccountName", $form),
-                    accountHolderType: bfjs.core.getFormValue("accountHolderType", $form),
-                    organizationID: self.transaction.bfjs.state.api.organizationID
-                };
+                payload = {};
+
+                if($form) {
```

Comment:
probably it is safer to use:
```js
if (self.transaction.formElementCandidate) {
```

because what does JQuery return for `$(undefined)`, `$(null)`, `$("")`? maybe it returns an empty array, which is `true`.